### PR TITLE
Make project more consistent & (hopefully) more SEO-friendly

### PR DESCRIPTION
**Pull request by tomschr**
* Rename DC file(s)
* Rename some XML file (art*.xml -> article*.xml)
* Rename some ID to use complete English words

Related to SUSE/doc-sleha#170

### CODE REVIEW

File: DC-SLES-SAP-sol-automation [hunk 1]
```diff
@@ -1,5 +1,5 @@
 MAIN="book_solution_based.xml"
-ROOTID="art-sol-automation"
+ROOTID="article-sol-automation"
```
Comment (taroth21) [inline DC-SLES-SAP-sol-automation:2]: maybe we can leave out the 'sol' (for solution here) and call the root ID 'article-sap-automation' (which would also be closer to the title?)

File: DC-SLES-SAP-sol-monitoring [hunk 2]
```diff
@@ -1,5 +1,5 @@
 MAIN="book_solution_based.xml"
-ROOTID="art-sol-monitoring"
+ROOTID="article-sol-monitoring"
```
Comment (taroth21) [inline DC-SLES-SAP-sol-monitoring:2]: maybe we can leave out the 'sol' (for solution here) and call the root ID 'article-sap-monitoring' (which would also be closer to the title?)

File: xml/book_solution_based.xml [hunk 3]
```diff
@@ -39,8 +39,8 @@ HINT (toms) 2020-06-05:
     <xi:include href="common_copyright_gfdl.xml"/>
   </info>
 
-  <xi:include href="art_solution_automation.xml"/>
-  <xi:include href="art_solution_monitoring.xml"/>
+  <xi:include href="article_solution_automation.xml"/>
```
Comment (taroth21) [inline xml/book_solution_based.xml:42]: same as above: should we rename to article_sap_automation.xml and  article_sap_monitoring.xml?

**Review by taroth21 — CHANGES_REQUESTED** [review]
Thanks! Please find my comments inline. As for the names of the DC files I would suggest:
DC-SLES-SAP-sol-automation -> DC-SLES-SAP-automation
DC-SLES-SAP-sol-monitoring -> DC-SLES-SAP-monitoring
DC-SLES-SAP-quick -> DC-SLES-SAP-installation

**Comment by tomschr** [comment]
Many thanks, I've adapted your suggestions.

**Pull request merged**